inference-engine/thirdparty/clDNN/src/graph_optimizer/set_onednn_optimization_attributes.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "pass_manager.h"
+#include "program_node.h"
+
+using namespace cldnn;
+
+void set_onednn_optimization_attributes::run(program& p) {
```

Comment:
I'd propose to add one more stage in the end of `prepare_primitive_fusing` pass which will initialize onednn post-ops and get rid of this standalone pass